features/SPS007_Spatial-containment.feature

```diff
@@ -0,0 +1,23 @@
+@implementer-agreement
+@SPS
+@version1
+@E00040
+
+Feature: SPS007 - Spatial Containment
+The rule verifies that correct spatial containment - IfcRelContainedInSpatialStructure
+
+    Scenario: Subtypes of IfcElement are placed within the project spatial hierarchy
+        Given an IfcSpatialElement
+        Given A relationship IfcRelContainedInSpatialStructure from IfcSpatialElement to IfcElement and following that
+        Given is_a != IfcElementAssembly
+        Given ContainedInStructure = not empty
+        Given its attribute ContainedInStructure
+        Given its relating entity
```

Comment:
```suggestion
        Given its attribute RelatingStructure
```

Can we use an existing step implementation?

Reply:
That's certainly an option. 
The intention was to create an implementation that is agnostic about the specific attribute used for lookup and to return only the 'relating' side of a relationship entity. Using the same tables as the statements _'A relationship X from/to Y from/to Z'_
In other words, regardless of the type of aggregate relationship entity—such as IfcRelAggregates or IfcRelContainedInSpatialStructure—it always retrieves the 'relating' side of that element. Or, the related side if you phrase it as 
`its related entities`

It also never harms to have multiple options to phrase gherkin I suppose .. 
What do you think?